docs/helm-chart/airflow-configuration.rst

```diff
@@ -68,3 +71,23 @@ configuration prior to installing and deploying the service.
   The recommended way to load example DAGs using the official Docker image and chart is to configure the ``AIRFLOW__CORE__LOAD_EXAMPLES`` environment variable
   in ``extraEnv`` (see :doc:`Parameters reference <parameters-ref>`). Because the official Docker image has ``AIRFLOW__CORE__LOAD_EXAMPLES=False``
   set within the image, so you need to override it when deploying the chart.
+
+..
+   Uncomment before merge
```

Comment:
Should probably do this then, shouldn't we :)

Reply:
Yes! As mentionned in the PR description, I am having issues cross-referencing documentation and would need help here.